tests/Integration/inc/Engine/Admin/Beacon/Beacon/insertScript.php

```diff
@@ -26,12 +26,15 @@ public function setUp() {
 	}
 
 	public function tearDown() {
-		parent::tearDown();
-
+		global $wp_locale;
 		set_current_screen( 'front' );
 
+		$wp_locale->text_direction = 'ltr';
```

Comment:
**[MINOR]** why not setting real current value on setup method and then use it here not to use static value.

Reply:
good point, I updated the test to reflect that